### PR TITLE
[ci2] add a log option to deploy steps

### PR DESCRIPTION
**Pull request by danking**
I think this is sufficient for our purposes

### CODE REVIEW

File: ci2/ci/build.py [hunk 1]
```diff
@@ -658,8 +660,17 @@ def config(self, deploy):  # pylint: disable=unused-argument
...
 
     async def cleanup(self, batch, deploy, sink):
-        # namespace cleanup will handle deployments
-        pass
+        script = ''
+        for tag in self.log:
+            script += f'kubectl -n {self.namespace} logs -l app={tag}\n'
+        script += 'date\n'
+        self.job = await batch.create_job(CI_UTILS_IMAGE,
+                                          command=['bash', '-c', script],
```
Comment (jigold) [inline ci2/ci/build.py]: This looks fine to me but where can I see the logs on the ui page?

Comment (danking) [reply ci2/ci/build.py]: They will be in the logs of a job called, for example, "deploy_batch_logs", which is an always run dependency of dependency of the sink

**Comment by danking** [comment]
Erm. I guess that's OK as long as we ensure the labels are the same as the service/deployment names.

**Comment by danking** [comment]
I see the existing code already make this assumption

**Comment by danking** [comment]
ok addressed cotton's comment as well.

**Comment by cseed** [comment]
> I see the existing code already make this assumption

Yes.

File: ci2/ci/build.py (hunk 1 above)
Comment (jigold) [reply ci2/ci/build.py]: I'm still not seeing it. Can you confirm you get the output you want or give me a link?

**Comment by danking** [comment]
@jigold you won't see the output on this job, this is a change to CI so that every PR that CI builds lists the logs. If the internal links worked and we had a test for ci2, then you'd be able to see it at the newly tested CI's website at: https://ci2.pr-6094-default-q0023anv9fmd.internal.hail.is, but neither do we test ci2 nor do these internal links seem to work.

**Comment by jigold** [comment]
Ok. Will approve then.

**Pull request merged**